lib/valkyrie/persistence/fedora/metadata_adapter.rb

```diff
@@ -15,13 +15,11 @@ class MetadataAdapter
     # @param [String] base_path
     # @param [Valkyrie::Persistence::Fedora::PermissiveSchema] schema
     # @param [Integer] fedora_version
-    def initialize(connection:, base_path: "/", schema: Valkyrie::Persistence::Fedora::PermissiveSchema.new, fedora_version: 4)
+    def initialize(connection:, base_path: "/", schema: Valkyrie::Persistence::Fedora::PermissiveSchema.new, fedora_version: 5)
```

Comment:
Use a constant in `fedora.rb`.

Reply:
Huh? Like a DEFAULT_FEDORA_VERSION? I'm not really sure what that gets us.